source/features/batch-mark-files-as-viewed.tsx

```diff
@@ -38,10 +41,28 @@ function batchToggle(event: delegate.Event<MouseEvent, HTMLFormElement>): void {
 	}
 }
 
+function markAsViewedSelector(target: HTMLElement): string {
+	const checked = (target as HTMLInputElement).checked ? ':not([checked])' : '[checked]';
+	return '.js-reviewed-checkbox' + checked;
+}
+
+const markAsViewed = clickAll(markAsViewedSelector);
+
+async function onAltClick(event: delegate.Event<MouseEvent, HTMLElement>): Promise<void> {
+	if (event.altKey && event.isTrusted) {
+		const lastCheckboxCompleted = oneMutation(select.last(markAsViewedSelector(event.delegateTarget))!, {attributes: true});
+		const hideToast = await showToast();
```

Comment:
We could customize the message, maybe:

```suggestion
		const message = event.delegateTarget.checked ?
			'Marking visible files as viewed' :
			'Marking visible files as unviewed';
		const hideToast = await showToast(message);
```

And also in `hideToast`